components/RecipeCard/RecipeCard.tsx

```diff
@@ -1,26 +1,34 @@
 'use client';
 
 import { Center, Text, Group, Flex, Stack } from '@mantine/core';
-import { GiForkKnifeSpoon, GiSaucepan } from 'react-icons/gi';
+import { GiForkKnifeSpoon } from 'react-icons/gi';
 import { Recipe } from '@/types/recipes';
+import CleanupRatingBanner from '@/components/Ratings/CleanupRatingBanner';
+import styled from 'styled-components';
 
 type RecipeCardProps = {
   recipe: Recipe;
 };
 
+const StyledContainer = styled.div`
```

Comment:
Should think about how to make some smaller reusable pieces - like this container that grows and the box shadow changes on hover - seems like something we use in a lot of places? But maybe not...